modules/sh-camera/modules/sh-camera/cameraUtils.js

```diff
@@ -0,0 +1,138 @@
+import { Alert, Platform } from 'react-native';
+import * as Permissions from 'react-native-permissions';
+import ImagePicker from 'react-native-image-crop-picker';
+import axios from "axios";
```

Comment:
Can we use `fetch` here instead? `axios` isn't included.
https://reactnative.dev/docs/network

Reply:
okay, Axios was already installed, looks like part of scaffolding, I didn't add it to dependencies either. it was available already so I just used it.